pages/report-hazard/style.css

```diff
@@ -23,3 +23,16 @@ img {
 	width: 10%;
 	height: auto;
 }
+
+.desktop{
+	display: none;
+}
+
+@media screen and (min-width: 800px){
```

Comment:
Nice work having the media queries take care of the images section, did you remove the prev section of the code? just double-check so we don't have code in two places doing the same

Reply:
done